build/ios/NitenTokens.swift

```diff
@@ -0,0 +1,88 @@
+    static let colorInterfaceNegativeExtraDark = UIColor(red: 0.71, green: 0.01, blue: 0.00, alpha:1)
+    static let colorInterfaceNegativeLight = UIColor(red: 0.99, green: 0.93, blue: 0.93, alpha:1)
+    static let colorInterfaceNegativeMedium = UIColor(red: 0.96, green: 0.60, blue: 0.60, alpha:1)
+    static let colorInterfacePositiveDark = UIColor(red: 0.15, green: 0.63, blue: 0.35, alpha:1)
+    static let colorInterfacePositiveExtraDark = UIColor(red: 0.00, green: 0.40, blue: 0.07, alpha:1)
+    static let colorInterfacePositiveLight = UIColor(red: 0.92, green: 0.98, blue: 0.95, alpha:1)
+    static let colorInterfacePositiveMedium = UIColor(red: 0.63, green: 0.91, blue: 0.75, alpha:1)
+    static let colorInterfaceShadeDark = UIColor(red: 0.86, green: 0.86, blue: 0.86, alpha:1)
+    static let colorInterfaceShadeExtraDark = UIColor(red: 0.70, green: 0.70, blue: 0.70, alpha:1)
+    static let colorInterfaceShadeLight = UIColor(red: 0.97, green: 0.97, blue: 0.97, alpha:1)
+    static let colorInterfaceShadeMedium = UIColor(red: 0.95, green: 0.95, blue: 0.95, alpha:1)
+    static let fontFamilyDefault = 'Source Sans Pro'
+    static let fontStyleQuote = italic
+    static let fontStyleWeightBold = 600
+    static let fontStyleWeightLight = 300
+    static let fontStyleWeightRegular = 400
+    static let opacityOpaque = 0.6
+    static let opacityTranslucent = 0.35
+    static let shadowOffsetBottom1 = CGSize(width: 0, height:2)
+    static let shadowOffsetBottom2 = CGSize(width: 0, height:4)
+    static let shadowOffsetBottom3 = CGSize(width: 0, height:12)
+    static let shadowOffsetTop1 = CGSize(width: 0, height:-2)
+    static let shadowOffsetTop2 = CGSize(width: 0, height:-4)
+    static let shadowOffsetTop3 = CGSize(width: 0, height:-12)
+    static let shadowRadius1 = 4
+    static let shadowRadius2 = 12
+    static let shadowRadius3 = 24
+    static let sizeBorderRadiusL = CGFloat(16.00)
+    static let sizeBorderRadiusM = CGFloat(8.00)
+    static let sizeBorderRadiusNone = CGFloat(0.00)
+    static let sizeBorderRadiusS = CGFloat(4.00)
+    static let sizeBorderWidthNone = CGFloat(0.00)
+    static let sizeBorderWidthThick = CGFloat(2.00)
+    static let sizeBorderWidthThin = CGFloat(1.00)
+    static let sizeFontL = CGFloat(32.00)
+    static let sizeFontM = CGFloat(24.00)
+    static let sizeFontS = CGFloat(20.00)
+    static let sizeFontXl = CGFloat(48.00)
+    static let sizeFontXs = CGFloat(16.00)
+    static let sizeFontXxl = CGFloat(60.00)
+    static let sizeFontXxs = CGFloat(14.00)
+    static let sizeSpacingL = CGFloat(48.00)
+    static let sizeSpacingM = CGFloat(36.00)
+    static let sizeSpacingS = CGFloat(24.00)
+    static let sizeSpacingXl = CGFloat(60.00)
+    static let sizeSpacingXs = CGFloat(16.00)
+    static let sizeSpacingXxl = CGFloat(84.00)
+    static let sizeSpacingXxs = CGFloat(12.00)
+    static let sizeSpacingXxxl = CGFloat(148.00)
+    static let sizeSpacingXxxs = CGFloat(8.00)
```

Comment:
a q se refere esse espaçamento?

Reply:
@BMilliet this is the minor spacing existing

Reply:
mas é espaço entre elemetos? marggin? ou é entre letras e linhas? isso q eu nao entendi, pq espaçamentos de constraints são Int.

Reply:
@BMilliet são espaçamentos entre elementos e para as margens